commands/train.py

```diff
@@ -99,7 +100,9 @@ def main(
         )
 
         kili_print(f"Training on job: {job_name}")
-        os.environ["WANDB_PROJECT"] = title + "_" + job_name
+
+        if not disable_wandb:
+            wandb_run = wandb.init(project=title + "_" + job_name, reinit=True)
```

Comment:
this variable is only assigned if not disable_wandb 
Maybe assign it as None otherwise to avoid unassignement problems